src/citrine/informatics/executions/__init__.py

```diff
@@ -0,0 +1,3 @@
+# flake8: noqa
```

Comment:
I added the `executions` subfolder to mirror the workflows pattern, but perhaps we don't need the extra layering for executions?